openedx/features/partners/models.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib.auth.models import User
 from django.db import models
+
 from model_utils.models import TimeStampedModel
```

Comment:
Since both these modules are pip installed modules, there won't be any empty line here as per pep-8